docs/scss/components/_component-status-table.scss

```
@@ -5,20 +5,20 @@
 }
 
 .component-status-list-container {
-  border: 1px solid #c7c7c7;
+  border: var(--rh-border-width-sm, 1px) solid var(--rh-color-border-subtle-on-light, #c7c7c7);
```

awesome change thank you!

IIRC, on the docs stylesheets, you don't need to specify the fallbacks, because the tokens are available on the root element, so like
```suggestion
  border: var(--rh-border-width-sm) solid var(--rh-color-border-subtle-on-light);
```

---

@bennypowers thanks for the callout. i've removed the fallbacks